lib/cmds/configuration_layouts.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const options = require('../common-yargs');
+
+exports.command = 'configuration_layouts <command>';
```

Comment:
Based on other command names, I would suggest using `configuration-layouts` or even just `layouts`.

Reply:
I'll change it to "layouts".